manifest.json

```diff
@@ -602,6 +602,16 @@
       "useCaseName": "Multi-step workflow"
     }
   ],
+  "rust(provided.al2)": [
+    {
+      "directory": "rust(provided.al2)/cookiecutter-aws-sam-hello-rust",
```

Comment:
The runtime name is `rust` we decided to append `(provided.al2)` in the key to help distinguish lambda supported runtime from custom runtime which are built on existing `(provided or provided.al2)`.  However this new nomenclature only extends to the keys in the manifest file. 
We that said kindly change the directory from `rust(provided.al2)/cookiecutter-aws-sam-hello-rust` to `"rust/cookiecutter-aws-sam-hello-rust"`.

Reply:
Done!